argo_rollouts/enable-support-for-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="enable-support-for-namespace-scoped-argo-rollouts-installation"]
+= Enable support for namespace-scoped Argo Rollouts installation
+:context: enable-support-for-namespace-scoped-argo-rollouts-installation
+
+toc::[]
+The {gitops-title} enables support for two modes of Argo Rollouts installation:
```

Comment:
```suggestion
{gitops-title} enables support for two modes of Argo Rollouts installations:
```
The names of products do not take articles (i.e., "OpenShift Container Platform", not "the OpenShift Container Platform".

Reply:
Fixed.